x/protorev/keeper/emit.go

```diff
@@ -0,0 +1,56 @@
+package keeper
+
+import (
+	"strconv"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v15/x/protorev/types"
+)
+
+// EmitBackrunEvent updates and emits a backrunEvent
+func EmitBackrunEvent(ctx sdk.Context, backrunEvent sdk.Event, inputCoin sdk.Coin, profit, tokenOutAmount sdk.Int) {
```

Comment:
Is there a reason we want to separate `EmitBackrunEvent` and `CreateBackrunEvent` instead of just wrapping it into a single function?